src/remediations/fifi.js

```diff
@@ -68,6 +69,30 @@ exports.sortSystems = function (systems, column = 'system_name', asc = true) {
     return _.orderBy(systems, column, (asc) ? 'asc' : 'desc');
 };
 
+
+// TODO: Replace this logic with logic in the remediations-consumer
+exports.updatePlaybookRunsStatus = function (playbook_runs) {
+    _.forEach(playbook_runs, run => {
+        if (_.some(run.executors, executor => executor.status === FAILURE)) {
+            run.status = FAILURE;
+        } else if (_.some(run.executors, executor => executor.status === CANCELED)) {
```

Comment:
might be cleaner to isolate the logic into its own function and use return upon finding the desired states instead of all the else clauses